Civi/Api4/Action/CaseSalesOrder/ContributionCreateAction.php

```diff
@@ -69,28 +69,30 @@ class ContributionCreateAction extends AbstractAction {
   public function _run(Result $result) { // phpcs:ignore
     $resultArray = $this->createContribution();
 
-    $result->exchangeArray($resultArray);
+    return $result->exchangeArray($resultArray);
   }
 
   /**
    * {@inheritDoc}
    */
   private function createContribution() {
     $priceField = $this->getDefaultPriceSetFields();
+    $createdContributions = 0;
 
     foreach ($this->salesOrderIds as $id) {
       $transaction = CRM_Core_Transaction::create();
       try {
         $contribution = $this->createContributionWithLineItems($id, $priceField);
         $this->linkCaseSalesOrderToContribution($id, $contribution['id']);
         $this->updateCaseSalesOrderStatus($id);
+        $createdContributions++;
       }
       catch (\Exception $e) {
         $transaction->rollback();
       }
     }
 
-    return [];
+    return ['created_contributions' => $createdContributions];
```

Comment:
I would rename these to `created_contributions_count` and `$createdContributionsCount`